pkg/openthread/Makefile

```diff
@@ -0,0 +1,34 @@
+PKG_NAME=openthread
+PKG_URL=https://github.com/openthread/openthread.git
+PKG_VERSION=fbfd76a990b81f007957e1bd774e51bce742e53e
+PKG_BUILDDIR ?= $(BINDIRBASE)/pkg/$(BOARD)/$(PKG_NAME)
+
+$(info Compile OpenThread for FTD device)
+OPENTHREAD_ARGS+= --enable-cli-app=ftd --enable-application-coap
+
+$(info $$OPENTHREAD_ARGS is [${OPENTHREAD_ARGS}])
+
+.PHONY: all
+
+all: git-download
+	cd $(PKG_BUILDDIR) && PREFIX="/" ./bootstrap
+	cd $(PKG_BUILDDIR) && ./configure \
+		CPP="$(CPP)" CC="$(CC)" CXX="$(CXX)" OBJC="" OBJCXX="" AR="$(AR)" RANLIB="$(RANLIB)" NM="$(NM)" \
+		STRIP="$(STRIP)" \
+		CPPFLAGS="-fdata-sections -ffunction-sections -Os -g  $(CFLAGS_CPU) " \
+		CFLAGS="-fdata-sections -ffunction-sections -Os -g  $(CFLAGS_CPU) " \
+		CXXFLAGS="-fdata-sections -ffunction-sections -Os -g  $(CFLAGS_CPU) -fno-exceptions -fno-rtti " \
+		LDFLAGS="-fdata-sections -ffunction-sections -Os -g  $(CFLAGS_CPU) -nostartfiles -specs=nano.specs \
+		-specs=nosys.specs -Wl,--gc-sections -Wl,-Map=map.map " \
+		--host=$(TARGET_ARCH) \
+		--target=$(TARGET_ARCH) \
+		--prefix=/ \
+		--enable-default-logging \
+		${OPENTHREAD_ARGS}
+	cd $(PKG_BUILDDIR) && make -j4 --no-print-directory DESTDIR=$(PKG_BUILDDIR)/output install PREFIX=/
```

Comment:
DESTDIR=${} and PREFIX=/ should be put before make -j4 to avoid warning such as:
```
openthread/third_party/nlbuild-autotools/repo/scripts/mkversion: line 133: arm-none-eabi-/dev/null: No such file or directory
```